src/Bridge/Symfony/Bundle/Test/ApiTestCase.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\Test;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+/**
+ * Base class for functional API tests.
+ *
+ * @experimental
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+abstract class ApiTestCase extends KernelTestCase
+{
+    /**
+     * Creates a Client.
+     *
+     * @param array $options An array of options to pass to the createKernel method
+     *
+     * @return Client A Client instance
+     */
+    protected static function createClient(array $options = [])
```

Comment:
Hi ! What do you think about moving this method in a trait and rename it `createApiClient` ? So we can use it with any other/custom TestCase.

Reply:
I'm not sure it is worth it. This method just accesses to a service from the container. It can easoly be copy/pasted/adapted.